pulp_rpm/tests/functional/constants.py

```diff
@@ -9,6 +9,10 @@
     CONTENT_PATH,
 )
 
+
+DOWNLOAD_POLICIES = ['cache_only', 'immediate', 'on_demand']
```

Comment:
s/cache_only/streamed/

Reply:
I just saw this renaming now. After I commented on Pulp-smash repo.